### ISSUE TITLE
Entering learn mode with fixed frequency

### ISSUE THREAD

**Issue by datibbaw** [ThomasTavernier/node-broadlink#7]
The official app allows for a fixed frequency to be set on the device before it enters learning mode. Is that something that this library could do as well? It would be super helpful for me personally, because my remotes don't support press & hold to emit signals at a rapid pace, but I do know the exact frequency at which they transmit data.

**Comment by ThomasTavernier** [comment]
I don't know anything about this fixed frequency, is it something that is also implemented in the python library?
If so, I can convert it, if not, it would be difficult for me to do anything.
If you create an implementation, feel free to submit it

**Comment by datibbaw** [comment]
Unfortunately that's also not available in the Python library; It might be a feature that was newly introduced in the RM4 Pro line-up ... I couldn't find this in the SDK documentation either, so it may take a while to reverse engineer this 😢

**Comment by tilleul** [comment]
Is this related ?
https://github.com/mjg59/python-broadlink/pull/613

**Comment by ismaelbelghiti** [comment]
Yes it has been implemented in python library, here is the Pull Request:
https://github.com/mjg59/python-broadlink/pull/613/files

In Typescript you can enter learning mode with a given frequency like this:
```
enterRFLearningMode = async (frequency: number) => {
    let payload: number[] = [];
    const buffer = Buffer.alloc(4);
    buffer.writeUInt32LE(Math.floor(frequency * 1000), 0); // equivalent of '<I' without using struct
    payload = Array.from(buffer);
    this.send(0x1B, payload);
}
```
